### PR TITLE
profile name layout update

### PR DESCRIPTION
**Pull request by godrei**

### CODE REVIEW

File: autoprovision/platform.go [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+
+// Const
+const (
+	IOS   Platform = "iOS"
+	TVOS  Platform = "tvOS"
+	MacOS Platform = "macOS"
+)
+
+// BundleIDPlatform ...
+func (p Platform) BundleIDPlatform() (*appstoreconnect.BundleIDPlatform, error) {
```
Comment (lpusok) [inline autoprovision/platform.go]: Did not find any references, unused?

File: autoprovision/certificatehelper.go [hunk 2]
```diff
@@ -21,6 +21,7 @@ var (
 	AppStore    DistributionType = "app-store"
 	AdHoc       DistributionType = "ad-hoc"
 	Enterprise  DistributionType = "enterprise"
+	Direct      DistributionType = "direct"
```
Comment (lpusok) [inline autoprovision/certificatehelper.go]: Could comment that it is unsupported and macOS specific, or remove for now.

File: autoprovision/platform.go (hunk 1 above)
Comment (godrei) [reply autoprovision/platform.go]: yes, removing

File: autoprovision/certificatehelper.go (hunk 2 above)
Comment (godrei) [reply autoprovision/certificatehelper.go]: remove every mac profile related code

**Pull request merged**